.github/pull_request_template.md

```diff
@@ -1,8 +1,11 @@
 ## Description
 
-Please edit this to include a summary of the change (what and why).
+Please edit this to include a summary of the change (what).
 Include screenshots if you modify the UI.
```

Comment:
We can use comments instead of plain text as a placeholder so one doesn't need to delete it when creating an PR. It's slightly annoying to always have to delete the placeholder text when opening a new PR.

```suggestion
<!--- Please edit this to include a summary of the change (what).  -->
<!--- Include screenshots if you modify the UI.  -->
```